perf_test/runHowitzer.sh

```diff
@@ -53,7 +53,7 @@ for TEST in render/*.js; do
 
   if [[ -z $K6_PROMETHEUS_REMOTE_URL ]]; then
     echo "No K6_PROMETHEUS_REMOTE_URL was set, running default k6"
-    k6 run $TEST --summary-export=$SUMMARY_FILE_JSON >> $RESULT_FILE 2>> $ERROR_FILE
+    ./k6_test/k6 run $TEST --summary-export=$SUMMARY_FILE_JSON >> $RESULT_FILE 2>> $ERROR_FILE
```

Comment:
It looks this would change the`No K6_PROMETHEUS_REMOTE_URL`  case to use the custom `k6_test/k6` ?